README.md

```diff
@@ -66,7 +66,7 @@ Check out a new branch for your work in this tutorial (we'll call this branch
 
 ``` shell
 $ git checkout -b tutorial-example
-$ npm run test
+$ npm run test-web
```

Comment:
Need to add this in after-test command as well in step 7

Reply:
There is no after test in snapshot command as user need change something in DOM manually.

Reply:
No, actually the command to run test in step 7 is `npm run test` that should be `npm run test-web`